### DISCUSSION TITLE
Can't proceed payment in Myanmar.

### DISCUSSION THREAD

**Discussion by k66l** [community/community#48714] [Copilot Conversations]
### Select Topic Area

Question

### Body

I can't make a transaction for my github copilot. I subscribed it before. But today, I can't even subscribe to use copilot. It shows this error. Any idea for fixing this?

![Screenshot 2023-02-27 at 12 59 01 PM Small](https://user-images.githubusercontent.com/44778897/221587729-6030e12b-5b0c-4361-bc7b-789bf9b6c8c4.jpeg)

**Comment by iamcjwilliams** [comment]
@BumminZ Have you tried another form of payment?

**Reply by k66l** [reply]
@iamcjwilliams Yeah, I had tried another payment method. 
It was fixed by Github team, by resetting my account payment. I thought it was some kind of error, the reason is that my account was failed in recent transaction. I thought they locked my account for purchasing products. Thanks for your time bro.

**Comment by k66l** [comment]
Thanks for all those who helped me. I have found a solution, you have to open ticket in Github customer support. Then they will reset your account. Have fun...
